packages/graphback-datasync/src/DataSyncPlugin.ts

```diff
@@ -131,7 +128,7 @@ export class DataSyncPlugin extends GraphbackPlugin {
       }
     });
 
-    if (this.config.useVersion) {
+    if (this.config?.modelUsesVersion[model.graphqlType.name] || false) {
```

Comment:
```suggestion
    if (this.config?.modelUsesVersion[model.graphqlType.name]) {
```

Not sure if that is needed. Is the `|| false` in case the key does not exist in `modelUsesVersion`? In that case the condition would be `if (false)`.